docs/DeveloperGuide.md

```diff
@@ -277,12 +278,37 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 | Priority | As a …​                                    | I want to …​                     | So that I can…​                                                        |
 | -------- | ------------------------------------------ | ------------------------------ | ---------------------------------------------------------------------- |
-| `* * *`  | new user                                   | see usage instructions         | refer to instructions when I forget how to use the App                 |
-| `* * *`  | user                                       | add a new person               |                                                                        |
-| `* * *`  | user                                       | delete a person                | remove entries that I no longer need                                   |
-| `* * *`  | user                                       | find a person by name          | locate details of persons without having to go through the entire list |
-| `* *`    | user                                       | hide private contact details   | minimize chance of someone else seeing them by accident                |
-| `*`      | user with many persons in the address book | sort persons by name           | locate a person easily                                                 |
+| `* *`  | new home tutor                                   | have adding information be intuitive and logical         | use the app without hassle                 |
+| `* *`  | new user                                       | have a quick start guide              | learn how to use the app                                                                      |
+| `* *`  | new home tutor                                       | follow a set syllabus for my clients                |                                    |
+| `* *`  | new home tutor                                   | be able to quickly create a student’s profile with his/her relevant information          | |
+| `* *`    | new home tutor                                      | track the performance of my students   | prove to their parents that they are improving and my effectiveness as a tutor                |
+| `* * *`     | new home tutor | track all my appointments          | I don’t forget or double book myself                                               |
```

Comment:
So that I can 'reduce chances of forgetting appointments or double-booking myself'